contrib/tripleo/skydive-tripleo-inventory

```diff
@@ -183,21 +184,31 @@ def parse_config():
 
 def generate_inventory(configs, inventory, path, extra_vars=None,
                        analyzer_extra_vars=None, agent_extra_vars=None,
-                       undercloud_params=None, host_network='ctlplane'):
+                       undercloud_params=None, container_params=None,
+                       host_network='ctlplane'):
     docker_registry = ''
     if undercloud_params:
         addresses = undercloud_params['parameter_defaults'].get(
             'DockerInsecureRegistryAddress')
         if addresses:
             docker_registry = addresses[0]
 
+    ns = 'rhosp15'
+    tag = 'latest'
+    if container_params:
+        for params in container_params['parameter_defaults'].get(
+             'ContainerImagePrepare', {}):
+            ns = params.get('set', {}).get('namespace',
+                                           '/rhosp15').split('/')[1]
+            tag = params.get('set', {}).get('tag', 'latest')
```

Comment:
how about using like this
''' tag = params.get('set', {}).get('tag', tag)'''
so that if we got some value in first iteration and empty set in second iteration we can use the first one instead of the default one.

Reply:
I completely agree with you. Unfortunately testing TripleO is such a PITA - roughly 6 hours - that I prefer saving some developper time (and precious resources...)